dataset_eval/evaluate.py

```diff
@@ -60,11 +60,15 @@ def evaluate_dataset(eval_job, dataset_dir, storage_dir):
         with open(groundtruth_path, "w") as f:
             yaml.dump(create_groundtruth_dict(snapshot["data"]["name"], train), f)
 
+        # Passing more user preferences to train the model.
```

Comment:
I don't think this comment is needed here really, but if it is, `Pass user preferences to train the model` would read better. Note alastair's comment about tense in comments.